src/components/screens/management-screen.tsx

```diff
@@ -0,0 +1,438 @@
+    }
+
+    const newDisplayedPersonsTotalTime = personsTotalTime.filter(personTotalTime =>
+      `${personTotalTime.person.firstName} ${personTotalTime.person.lastName}`.toLowerCase().includes(newSearchInput.toLowerCase()));
+
+    setDisplayedPersonsTotalTime(newDisplayedPersonsTotalTime);
+  };
+
+  /**
+   * Renders the search 
+   */
+  const renderSearch = () => {
+    return (
+      <Box className={ classes.searchContainer }>
+        <TextField
+          value={ searchInput }
+          onChange={ handleSearchInputChange }
+          placeholder={ strings.managementScreen.searchPlaceholder }
+          variant="outlined"
+          className={ classes.searchTextField }
+          inputProps={{ style: { paddingLeft: theme.spacing(5) } }}
+        />
+        <SearchIcon className={ classes.searchIcon }/>
+      </Box>
+    );
+  };
+
+  /**
+   * Renders the redirect link 
+   */
+  const renderRedirect = () => {
+    return (
+      <Link to="/">
+        <Paper className={ classes.redirectLinkPaper }>
+          <SubdirectoryArrowLeftIcon fontSize="large"/>
+        </Paper>
+      </Link>
+    );
+  };
+
+  /**
+   * Renders the customized label for charts
+   * 
+   * @param props props of the custom label
+   */
+  const renderCustomizedLabel = (props: CustomPieLabel) => {
+    return TimeUtils.convertToMinutesAndHours(props.value);
+  };
+
+  /**
+   * Renders the customized tooltip for charts
+   * 
+   * @param props props of the custom tooltip
+   */
+  const renderCustomizedTooltip = (props: TooltipProps<ValueType, NameType>) => {
+    const { active, payload } = props;
+
+    if (!active || !payload || !payload.length) {
+      return null;
+    }
+
+    const selectedData = payload[0];
+
+    if (!selectedData.value || !selectedData.name) {
+      return null;
+    }
+
+    const sectionName = {
+      [WorkTimeCategory.PROJECT]: strings.project,
+      [WorkTimeCategory.INTERNAL]: strings.internal
+    }[selectedData.name];
+
+    return (
+      <Box style={{ backgroundColor: "rgba(0, 0, 0)" }}>
+        <Typography
+          variant="h6"
+          style={{
+            color: "#fff",
+            padding: theme.spacing(1)
+          }}
+        >
+          { `${sectionName}: ${TimeUtils.convertToMinutesAndHours(selectedData.value as number)}` }
+        </Typography>
+      </Box>
+    );
+  };
+
+  /**
+   * Renders the Total work time section
+   * 
+   * @param name name of the row
+   * @param value value of the row
+   * @param color color of the row value
+   */
+  const renderExpectedWorkRow = (name: string, value: string, color?: string) => {
+    return (
+      <Box className={ classes.expectedWorkRow }>
+        <Typography className={ classes.expectedWorkNames }>
+          { name }
+        </Typography>
+        <Typography
+          style={{ color: color }}
+          className={ classes.expectedWorkValues }
+        >
+          { value }
+        </Typography>
+      </Box>
+    );
+  };
```

Comment:
Simple renders like this can use the () => (); syntax instead of () => { return ... };